packages/actor-scraper/web-scraper/INPUT_SCHEMA.json

```diff
@@ -186,7 +186,7 @@
         "maxCrawlingDepth": {
             "title": "Max crawling depth",
             "type": "integer",
-            "description": "Specifies how many links away from <b>Start URLs</b> the scraper will descend. This value is a safeguard against infinite crawling depths for misconfigured scrapers. Note that pages added using <code>context.enqueuePage()</code> in <b>Page function</b> are not subject to the maximum depth constraint. <br><br>If set to <code>0</code>, there is no limit.",
+            "description": "Specifies how many links away from <b>Start URLs</b> the scraper will descend. This value is a safeguard against infinite crawling depths for misconfigured scrapers. Note that pages added using <code>context.enqueuePage()</code> in <b>Page function</b> are not subject to the maximum depth constraint. <br><br>If set to <code>0</code>, there is no limit. To crawl only the pages specified by the Start URLs, set <a href=\"#linkSelector\"><code>linkSelector</code></a> empty instead.",
```

Comment:
But if it's empty in input UI, doesn't the actor run with the default value?

Reply:
Not sure how this works on the platform, but there is no default value in the input schema for the `linkSelector` either way. I just tried it out, it runs as expected (not enqueueing anything).